config/application.rb

```diff
@@ -13,10 +13,11 @@
 
 module PrisonVisits
   class Application < Rails::Application
+    config.active_record.schema_format = :sql
     config.phase = 'live'
     config.product_type = 'service'
 
-    config.autoload_paths += %w[ app/mailers/concerns ]
+    config.autoload_paths += %w[ app/mailers/concerns app/metrics ]
```

Comment:
Is this definitely necessary? I thought all paths under `app` that weren't already special were automatically included in autoload paths.